### ISSUE TITLE
Missing static directory for webapp

### ISSUE THREAD

**Issue by jronnblom** [keylime/keylime-deb#3]
The static directory is missing from /usr/lib/python3/dist-packages/keylime

**Comment by galmasi** [comment]
Yep. We have not been running webapp because it has been not working for a while.
